src/ir_Mitsubishi.cpp

```diff
@@ -1186,3 +1198,449 @@ String IRMitsubishi136::toString(void) {
+                               true, _tolerance, 0, false);
+  if (!used) return false;
+  if (strict) {
+    // Header validation: Codes start with 0x23CB26
+    if (results->state[0] != 0x23 || results->state[1] != 0xCB ||
+        results->state[2] != 0x26) return false;
+    if (!IRMitsubishi112::validChecksum(results->state, nbits / 8))
+      return false;
+  }
+  results->decode_type = MITSUBISHI112;
+  results->bits = nbits;
+  return true;
+}
+#endif  // DECODE_MITSUBISHI112
+// Code to emulate Mitsubishi 112bit A/C IR remote control unit.
+//
+// Equipment it seems compatible with:
+//   Brand: Mitsubishi Electric,  Model: PEAD-RP71JAA Ducted A/C
+//   Brand: Mitsubishi Electric,  Model: 001CP T7WE10714 remote
+
+// Initialise the object.
+IRMitsubishi112::IRMitsubishi112(const uint16_t pin, const bool inverted,
+                                 const bool use_modulation)
+    : _irsend(pin, inverted, use_modulation) { this->stateReset(); }
+
+// Reset the state of the remote to a known good state/sequence.
+void IRMitsubishi112::stateReset(void) {
+  // The state of the IR remote in IR code form.
+  // Known good state obtained from:
+  remote_state[0] = 0x23;
+  remote_state[1] = 0xCB;
+  remote_state[2] = 0x26;
+  remote_state[3] = 0x01;
+  remote_state[4] = 0x00;
+  remote_state[5] = 0x24;
+  remote_state[6] = 0x03;
+  remote_state[7] = 0x0B;
+  remote_state[8] = 0x10;
+  remote_state[9] = 0x00;
+  remote_state[10] = 0x00;
+  remote_state[11] = 0x00;
+  remote_state[12] = 0x30;
+  remote_state[13] = 0x75;
+
+  checksum();  // Calculate the checksum which covers the rest of the state.
+}
+
+// Calculate the checksum for the current internal state of the remote.
+void IRMitsubishi112::checksum(void) {
+  uint8_t checksum = 0;
+  for (uint8_t i = 0; i < 13; i++) {
+      checksum = checksum + remote_state[i];
+  }
+  remote_state[13] = checksum;
+}
+
+bool IRMitsubishi112::validChecksum(const uint8_t *data, const uint16_t len) {
+  uint8_t checksum = 0;
+
+  if (len < kMitsubishi112StateLength) return false;
+
+  for (uint8_t i = 0; i < kMitsubishi112StateLength-1; i++) {
+    checksum += data[i];
+  }
+
+  if (data[kMitsubishi112StateLength-1] != checksum) return false;
+
+  return true;
+}
+
+// Configure the pin for output.
+void IRMitsubishi112::begin(void) { _irsend.begin(); }
+
+#if SEND_MITSUBISHI112
+// Send the current desired state to the IR LED.
+void IRMitsubishi112::send(const uint16_t repeat) {
+  checksum();  // Ensure correct checksum before sending.
+  _irsend.sendMitsubishi112(remote_state, kMitsubishi112StateLength, repeat);
+}
+#endif  // SEND_MITSUBISHI112
+
+// Return a pointer to the internal state date of the remote.
+uint8_t *IRMitsubishi112::getRaw(void) {
+  checksum();
+  return remote_state;
+}
+
+void IRMitsubishi112::setRaw(const uint8_t *data) {
+  for (uint8_t i = 0; i < (kMitsubishi112StateLength - 1); i++) {
+    remote_state[i] = data[i];
+  }
+  checksum();
```

Comment:
Again, as per earlier, it shouldn't be needed. You've got it covered in `setRaw()` and `send()` already.